cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -58,14 +63,17 @@
             </div>
         </div>
     </fieldset>
+    {% else %}
+    <input type="hidden"
+           name="is_helpful"
+           value="{{ request.GET.get('is_helpful', '') }}">
```

Comment:
Instead of passing the request query parameter into the page like this, does it make sense to check the input (either through some logic here, or in a helper method) and force a value of `1` or `0`?